packages/react-ui-map/src/Mapbox/Source.js

```diff
@@ -24,16 +27,6 @@ export default class Source extends Component {
     map: PropTypes.object,
   }
 
-  componentWillUnmount() {
```

Comment:
Do we not still want this?

Reply:
Layers need to be removed before sources or else there will be an error.  I thought it made more sense to go the ag.js route and only add the source in the constructor if it doesn't already exist, instead of trying to remove it explicitly.

Reply:
Ok, sounds like a good approach.